### ISSUE TITLE
crash

### ISSUE THREAD

**Issue by aFadilnashshad** [nuclearsilo583/zephyrus-store-preview-new-syntax#84]
server crash when i load store.smx , when i disable it , it doesnt crash , when i re add all files, its ok for 1 round, than i restart it crashes. ill send log here ->  L 07/12/2022 - 08:53:51: [store.smx] SQL error happened.
Query: DELETE FROM store_logs WHERE `date` < (SELECT DATETIME('now', '-7 day'))
Error: no such table: store_logs

this inly error i get from store ? but it crahes and i know it store becuase if i disbale , it doesnt crash.
but i also disabled other store plugins that i dont want, is it crash becuase they are not loaded?

**Comment by aFadilnashshad** [comment]
yep still this error. L 07/12/2022 - 09:12:27: [store.smx] SQL error happened.
Query: DELETE FROM store_logs WHERE `date` < (SELECT DATETIME('now', '-7 day'))
Error: no such table: store_logs 
and yesterday i updated to latest version u added

**Comment by nuclearsilo583** [comment]
Can you give me your plugin list? And is there any error log that show up.
the table store_logs was not created when you're using sqlite, this mistake was from myside and will be remove on next commit.

**Comment by aFadilnashshad** [comment]
sure here,
admin-flatfile.smx

adminhelp.smx

adminmenu.smx

advertisements.smx

ambient.smx
antiflood.smx

autozspawn.smx

basebans.smx

basechat.smx

basecomm.smx

basecommands.smx
basetriggers.smx
basevotes.smx

bosshud.smx

bot_spawner.smx

bspconvar_custom_whitelist.smx

buttonwatcher.smx

chat-processor.smx

clientprefs.smx

convarsettings.smx

countdown_detector.smx

csgo_movement_unlocker.smx

entwatch_csgo_dz.smx

filesmanagementinterface.smx

fpvm_interface.smx

franug_colors.smx

franug_noagentsmodels.smx

franug_weapons.smx

funcommands.smx

funvotes.smx

gifts.smx

gun_menu.smx

hide_teammates.smx

knockbackfix.smx

mapchooser.smx

mapmusic_dz.smx

n_arms_fix.smx


nextmap.smx

noblock.sm
nominations.smx
8
NoShake.smx
1
playercommands.smx

reservedslots.smx

rockthevote.smx
shownamehud_white.smx

soundlib_demo.smx

sounds.smx
store_item_admin.smx
store_item_bullet_impact/smx
store_item_bulletsparks.smx
store_item_bunnyhop.smx
store_item_colored_smoke.smx
store_item_cpsupport.smx
store_item_grenskins.smx
store_item_grentrails.smx
store_item_hats_no_thirdperson.smx
store_item_info.smx
store_item_lasersight.smx
store_item_link.smx
store_item_mvpsound.smx
store_item_particle.smx
store_item_pet.smx
store_item_playerskins.smx
store_item_saysound.smx
store_item_sprays.smx
store_item_tracers.smx
store_item_trails.smx
store_item_weaponmodel.smx
store_misc_dosh.smx
store_misc_earnings.smx
store_misc_giveaway_aidnversion.smx

store_misc_giveaway.smx
store_misc_math.smx
store_misc_toplists.smx
store_misc_voucher.smx
store-trade.smx
thirdperson_csgo.smx
thirdperson.smx
toggle_weapon_sounds.smx
togsc.smx
top_defenders.smx
zombiereloaded_sounds.smx

here you go ,also when next commit?

**Comment by nuclearsilo583** [comment]
First please install [accelerator report](https://forums.alliedmods.net/showthread.php?t=277703) for future logging about the crash. It's must be the player model or any model that was not precache properly on map start. Please install one of [these](https://forums.alliedmods.net/showthread.php?p=602270) [plugin](https://forums.alliedmods.net/showthread.php?t=269792) .

**Comment by aFadilnashshad** [comment]
oh yeas i had errors about models prechaching but ti was aboput ambient.mp3 and the phenoix model, thanks about this :)
